setup.py

```diff
@@ -1,14 +1,25 @@
+"""
+    The setup file.
+"""
+import os
 import setuptools
 
-setuptools.setup(
-    install_requires=[
-        "stable-diffusion-sdkit", # wrapper around stable-diffusion, to allow pip install
+install_requires = []
+
+
+if not os.getenv('SKIP_EXTERNAL_DEPS'):
```

Comment:
Can you please explain why we need to check this variable? And what's the use-case for installing sdkit without the dependencies?